flowmachine/flowmachine/features/network/total_network_objects.py

```diff
@@ -249,39 +238,9 @@ class AggregateNetworkObjects(GeoDataMixin, Query):
 
     """
 
-    def __init__(
-        self,
-        start=None,
-        stop=None,
-        statistic="avg",
-        table="all",
-        period="day",
-        by=None,
-        network_object="cell",
-        level="admin0",
-        column_name=None,
-        size=None,
-        polygon_table=None,
-        geom_col="geom",
-        hours="all",
-        subscriber_subset=None,
-        subscriber_identifier="msisdn",
-    ):
-        self.total_objs = TotalNetworkObjects(
-            start=start,
-            stop=stop,
-            table=table,
-            period=period,
-            network_object=network_object,
-            level=level,
-            column_name=column_name,
-            size=size,
-            polygon_table=polygon_table,
-            geom_col=geom_col,
-            hours=hours,
-            subscriber_subset=subscriber_subset,
-            subscriber_identifier=subscriber_identifier,
-        )
+    def __init__(self, total_objs, statistic="avg", by=None):
+        self.total_objs = copy.deepcopy(total_objs)
```

Comment:
Don't need a deepcopy here, just an assign is fine.

Reply:
Below this, period will now need to refer to the period attribute of the passed in object.